src/components/Authenticate.jsx

```diff
@@ -0,0 +1,192 @@
+                                    required
+                                />
+                            </div>
+                            
+                            {/* <div className="form-group text-center">
+                                {status === "SUCCESS" ? (
+                                    <h5 className="text-success">
+                                        Thanks! Your response has been submitted
+                                    </h5>
+                                ) : (
+                                    <button
+                                        type="submit"
+                                        className="btn btn-outline-success rounded-pill shadow"
+                                        style={{width: "30%"}}
+                                    >
+                                        Send
+                                    </button>
+                                )}
+                                {status === "ERROR" && (
+                                    <h5 className="text-danger">Oops! There was an error.</h5>
+                                )}
+                            </div> */}
+                            <div className="form-group text-center">
+                              <button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                style={{width: "30%"}}
+                              >
+                                SignIn
+                              </button>
+                              <button
+                                type="submit"
+                                className="btn btn-outline-danger rounded-pill shadow"
+                                style={{width: "50%",marginLeft: "5px"}}
+                              >
+                                Forgot your password?
+                              </button>
+                            </div>
+                            <div className="form-group text-center">
+                              <label className="h5 contact-label">New to Social-Scheduler ? ,then Register here</label>
+                              <button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                style={{width: "30%"}}
+                              >
+                                SignUp
+                              </button>
+                            </div>
+                            {/* social media links for login */}
+                            <div>
+                              <div style={{margin:"40px"}}>
+                                <h2><span>OR</span></h2>
+                              </div>
+                              <div className="col">
+                                <ul style={{display:"inline",listStyle:"none"} }>
```

Comment:
Use styled components, Instead of inline CSS

Reply:
how can you make a styled compoent, instead of inline css, there is no styled component like that right?

Reply:
You have to create custom styled component.